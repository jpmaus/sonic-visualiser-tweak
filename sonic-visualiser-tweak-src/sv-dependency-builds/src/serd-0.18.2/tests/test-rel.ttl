<http://www.w3.org/2001/sw/DataAccess/df1/tests/a/b/c> a <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1> a <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/test-rel.ttl> a <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/test-not.ttl> a <http://example.org/Thing> .
<a/b/c> a <http://example.org/Thing> .
<http://www.w3.org/> a <http://example.org/Thing> .
