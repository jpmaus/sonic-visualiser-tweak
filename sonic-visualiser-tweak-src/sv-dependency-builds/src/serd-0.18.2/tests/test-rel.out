<http://www.w3.org/2001/sw/DataAccess/df1/tests/a/b/c> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/test-rel.ttl> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/test-not.ttl> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/a/b/c> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
<http://www.w3.org/> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/Thing> .
